src/csi/pkg/util/nvmf.go

```diff
@@ -123,7 +123,7 @@ func deleteSubsystem(params map[string]string, mtx2 *sync.Mutex) error {
 			"subnqn": "%s"
 		}
 	}`, params["nqn"]))
-	url := fmt.Sprintf("http://%s:%s/api/ibofos/v1/subsystem", params["provisionerIP"], params["provisionerPort"])
+	url := fmt.Sprintf("http://%s:%s/api/ibofos/v1/subsystem", params["provisionerIp"], params["provisionerPort"])
```

Comment:
It also needs to be changed to "provisionerIP".

Reply:
Same reply above